Makefile.common

```diff
@@ -6,7 +6,7 @@
 # The default build target: just build the scummvm executable
 ######################################################################
 
-all: $(EXECUTABLE) plugins
+all: $(EXECUTABLE) plugins readme
```

Comment:
This causes a compilation error when  pandoc is not present on the system, of if `--enable-pandoc` is not used when running configure:  `make: *** No rule to make target 'readme', needed by 'all'.  Stop.`

(note: due to another issue - see comment below - we actually currently get yet another issue here, but fixing the other issue then result in the error above)

I think you need here:
```
all: $(EXECUTABLE) plugins

ifdef USE_PANDOC
all: readme
endif
```

Reply:
Fixed